<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderDlg</class>
 <widget class="QWidget" name="OrderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>212</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblStkSymbol">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Symbol Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblQuantity">
        <property name="text">
         <string>Quantity : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="cmbPriceType">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>MARKET</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LIMIT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SL-M</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbQuantity">
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dsbPriceValue"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblAvailQuantity">
        <property name="text">
         <string>Available Quantity : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblAvailQuantityVal">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnOrder">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
